torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -55,6 +66,20 @@ function HudRow({
   const router = useRouter();
   const params = packHudParams(router.query);
   const sha = rowData.sha;
+
+  const failedJobs = rowData.jobs.filter(isFailedJob);
+  const { repoOwner, repoName } = router.query;
+
+  const { data } = useSWR(
+    `/api/job_annotation/${repoOwner}/${repoName}/annotations/${encodeURIComponent(
+      JSON.stringify(failedJobs.map((failedJob) => failedJob.id))
+    )}`,
+    fetcher
+  );
+  if (data == null) {
+    return null;
```

Comment:
Why return null if there are no classified jobs instead of returning what's returned on the page today?

Reply:
returning null here means that we don't show it until we fetch the data.